### DISCUSSION TITLE
Graph

### DISCUSSION THREAD

**Discussion by aNNaSkmOcc** [maximevaillancourt/digital-garden-jekyll-template#132] [Q&A] [answered]
Hey Guys,

First of thank you for this amazing template.

My question is: How can i remove the graph ?

**Comment by maximevaillancourt** [comment, marked as answer]
You may remove it by removing these lines:

https://github.com/maximevaillancourt/digital-garden-jekyll-template/blob/e6e2ea29321aa9ab7e271de943e80ca264ae3dc2/_layouts/note.html#L42-L46

**Reply by abdelkareemkobo** [reply]
this link  is not working anymore

**Reply by maximevaillancourt** [reply]
Thank you for pointing this out! Link updated. ✅
